spec/features/finance/payment_breakdown_spec.rb

```diff
@@ -22,15 +23,29 @@
     and_i_click_the_submit_button
     and_i_click_open_all_button
     then_the_page_should_be_accessible
-    and_percy_should_be_sent_a_snapshot_named("Payment breakdown for an ECF provider")
+    and_percy_should_be_sent_a_snapshot_named("Payment breakdown for an ECF provider (eligible)")
 
     when_i_click_on_view_contract_link
     then_the_page_should_be_accessible
     and_percy_should_be_sent_a_snapshot_named("Contract breakdown for an ECF provider")
   end
 
+  scenario "Can get to ECF payment breakdown page for payable declarations" do
+    given_i_am_logged_in_as_a_finance_user
+    and_there_is_ecf_provider_with_contract
+    and_there_is_a_schedule
+    visit "/finance/ecf/payment_breakdowns/cffd2237-c368-4044-8451-68e4a4f73369/payable"
```

Comment:
```suggestion
    visit "/finance/ecf/payment_breakdowns/cffd2237-c368-4044-8451-68e4a4f73369/payable"
```

Can you share this ID across the spec?

Reply:
I've rolled this test into the previous one to remove the overhead of performing the setup again. also no need to do the `visit`